#162948801 Create UI for registration

andrewhingah opened the pull request:
**What does this PR do?**
- creates UI for user registration

**Description of tasks to be completed**
- create Signup form component
- render the Signup form component
- style signup form with Bootstrap 4

**Relevant PT stories**
[#162948801](https://www.pivotaltracker.com/story/show/162948801)

**Screenshot**

![image](https://user-images.githubusercontent.com/32644295/52839802-565dd480-3108-11e9-878e-afdb233e4667.png)

andrewhingah committed:
feat(registration) create UI for registration

- add signup form
- style the form with boostrap
- provide social auth buttons

[Delivers: #162948801]

cosmas28 commented:
Kindly fix the Travis first

cosmas28 commented:
- Kindly Fix the typos in the `<scope>` section of your commit message.
- Fix merge conflicts.

cosmas28 commented on src/components/Signup/index.jsx:
```diff
@@ -0,0 +1,69 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './signup.scss';
+
+const SignupForm = ({
```
The component name should the same to the component directory name.

cosmas28 commented on src/components/Signup/index.jsx:
```diff
@@ -0,0 +1,69 @@
+                        <h4 className='card-title mt-2'>Create Account</h4>
+                    </header>
+                    <article className='card-body'>
+
```
Kindly remove these unnecessary line spaces from this form.

cosmas28 reviewed (changes requested):
Good work, otherwise add tests.

cosmas28 commented:
- The form is awesome, otherwise, the Signup page is a half screen. What do you think about making the page to fit the screen?

- Also, write tests for the form and the page.

cosmas28 commented on src/components/Signup/index.jsx:
```diff
@@ -0,0 +1,63 @@
+import './signup.scss';
+
+const Signup = ({
+    handleChange, handleSubmit,
```
The destructuring is dope...keep up the good work.

cosmas28 commented on src/components/Signup/signup.scss:
```diff
@@ -0,0 +1,3 @@
+.container{
```
The `.scss` file name should the same to it's component name.

cosmas28 commented on src/pages/Signup/index.jsx:
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import SignupForm from '../../components/Signup';
+
+class SignUp extends Component {
```
The component name should be the same to its directory name.

cosmas28 commented on src/routes/Routes.test.js:
```diff
@@ -11,8 +11,4 @@ describe('Routes', () => {
     expect(wrapper.find({ path: '/' }).length).toBe(1);
   });
-
-  it('should render the login route', () => {
```
Any reason why you removed this test?

andrewhingah committed:
feat(registration): add test for signup route and snapshot

- add test for signup route
- add test for signup snapshot

[Delivers #162948801]

cosmas28 commented on src/pages/Signup/Signup.test.js:
```diff
@@ -0,0 +1,12 @@
+describe('Routes', () => {
+
+   
+    it('should render the signup route', () => {
```
too many lines before this `it` block.
